contracts/contracts/coordination/Coordinator.sol

```diff
@@ -95,17 +116,31 @@ contract Coordinator is Ownable {
         }
     }
 
+    function makeInitiationPublic() external onlyRole(DEFAULT_ADMIN_ROLE) {
+        isInitiationPublic = true;
+        _setRoleAdmin(INITIATOR_ROLE, bytes32(0));
+    }
 
-    function setTimeout(uint32 newTimeout) external onlyOwner {
+    function setTimeout(uint32 newTimeout) external onlyRole(DEFAULT_ADMIN_ROLE) {
         emit TimeoutChanged(timeout, newTimeout);
         timeout = newTimeout;
     }
 
-    function setMaxDkgSize(uint32 newSize) external onlyOwner {
+    function setMaxDkgSize(uint16 newSize) external onlyRole(DEFAULT_ADMIN_ROLE) {
         emit MaxDkgSizeChanged(maxDkgSize, newSize);
         maxDkgSize = newSize;
     }
 
+    function setReimbursementPool(IReimbursementPool pool) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (address(pool) == address(0)){
+            delete reimbursementPool;
+        } else {
+            require(pool.isAuthorized(address(this)), "Coordinator not authorized");
```

Comment:
for me looks like it could be optimized slightly (unless I'm missing something):
`require(address(pool) == address(0) || pool.isAuthorized(address(this))`
instead of delete explicitly, just assign zero address. wdyt?

Reply:
✅